src/actions/admin.test.js

```diff
@@ -1037,4 +1037,30 @@ describe('Actions.Admin', () => {
         assert.ok(groups[key].mattermost_group_id === null);
         assert.ok(groups[key].has_syncables === null);
     });
+
+    it('getOAuthAppInfo', async () => {
+        const clientId = 'appId';
+        nock(Client4.getOAuthAppRoute(clientId)).
+            get('/info').
+            reply(200, OK_RESPONSE);
+
+        const {data, error} = await store.dispatch(Actions.getOAuthAppInfo(clientId));
+
+        assert.ok(data);
+        assert.deepStrictEqual(data, OK_RESPONSE);
```

Comment:
this test expects the status of the response to be 200 but nothing about the data?

Reply:
Yes, the idea is to check that anything server returns passes through the action without being changed. First I did replicate data structure in the response, but at the end we were checking only if they match, so this is the reasong why I changed it to a simpler object `OK_RESPONSE`

Reply:
I'd prefer to fake a real response and use that here. While it doesn't make much any difference to the actual behaviour of the test, I think mocking real responses is a good habit to have. It also makes it so that the tests don't immediately break if someone wants to have the action actually dispatch something based on the response.

Reply:
It makes sense to me, I will update it